ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -244,6 +244,14 @@
       "Config to enable full table scans for pull queries";
   public static final boolean KSQL_QUERY_PULL_TABLE_SCAN_ENABLED_DEFAULT = false;
 
+  public static final String KSQL_QUERY_PULL_INTERPRETER_ENABLED
+      = "ksql.query.pull.interpreter.enabled";
+  public static final String KSQL_QUERY_PULL_INTERPRETER_ENABLED_DOC =
+      "Enables whether we use the interpreter for expression evaluation for pull queries, or the"
+          + "default code generator. They should produce the same results, but the interpreter is"
+          + " much faster for short-lived queries.";
+  public static final boolean KSQL_QUERY_PULL_INTERPRETER_ENABLED_DEFAULT = false;
```

Comment:
do all of our tests pass if this is set to `true`?

Reply:
Yes, they do.  I just tested it and they all pass.